lib/WatcherProcess.php

```diff
@@ -275,7 +275,7 @@ protected function recommendLogLevel(Console $console) {
         if (!isset(Logger::LEVELS[$level])) {
             return;
         }
-        if (Logger::LEVELS[$level] > Logger::LEVELS[Logger::WARNING]) {
+        if (Logger::LEVELS[$level] > Logger::LEVELS[Logger::DEBUG]) {
             $this->logger->warning(
                 "Running aerys in production with a log level higher than \"warning\" is not " .
```

Comment:
Should probably adjust this `\"warning\"` too :)

Reply:
Ho my bad... yes of course... 👼